### PR TITLE
chore(delegation-api): Add missing infra config.

### PR DESCRIPTION
**Pull request by valurefugl**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

## Release Notes

- **New Features**
	- Integrated support for National Registry B2C services across multiple components, enhancing authentication and authorization capabilities.
	- Added new environment variables for National Registry B2C, including client ID, endpoint, path, and scope.

- **Bug Fixes**
	- Standardized health check configurations for improved service monitoring.

- **Chores**
	- Adjusted resource allocations for better performance and resource management.

### CODE REVIEW

File: charts/services/services-auth-delegation-api/values.prod.yaml [hunk 1]
```diff
@@ -28,6 +28,10 @@ env:
   IDENTITY_SERVER_CLIENT_ID: '@island.is/clients/auth-api'
   IDENTITY_SERVER_ISSUER_URL: 'https://innskra.island.is'
   LOG_LEVEL: 'info'
+  NATIONAL_REGISTRY_B2C_CLIENT_ID: '8271bbc2-d8de-480f-8540-ea43fc40b7ae'
+  NATIONAL_REGISTRY_B2C_ENDPOINT: 'https://skraidentity.b2clogin.com/skraidentity.onmicrosoft.com/b2c_1_midlun_flow/oauth2/v2.0/token'
+  NATIONAL_REGISTRY_B2C_PATH: 'IS/GOV/6503760649/SKRA-Cloud-Protected/Midlun-v1'
+  NATIONAL_REGISTRY_B2C_SCOPE: 'https://skraidentity.onmicrosoft.com/midlun/.default'
```
[the remark this answers is not in the document]
Comment (valurefugl) [reply charts/services/services-auth-delegation-api/values.prod.yaml:34]: not secrets

File: charts/services/services-auth-delegation-api/values.dev.yaml [hunk 2]
```diff
@@ -28,6 +28,10 @@ env:
   IDENTITY_SERVER_CLIENT_ID: '@island.is/clients/auth-api'
   IDENTITY_SERVER_ISSUER_URL: 'https://identity-server.dev01.devland.is'
   LOG_LEVEL: 'info'
+  NATIONAL_REGISTRY_B2C_CLIENT_ID: '6cf94113-d326-4e4d-b97c-1fea12d2f5e1'
+  NATIONAL_REGISTRY_B2C_ENDPOINT: 'https://skraidentitydev.b2clogin.com/skraidentitydev.onmicrosoft.com/b2c_1_midlun_flow/oauth2/v2.0/token'
+  NATIONAL_REGISTRY_B2C_PATH: 'IS-DEV/GOV/10001/SKRA-Cloud-Protected/Midlun-v1'
+  NATIONAL_REGISTRY_B2C_SCOPE: 'https://skraidentitydev.onmicrosoft.com/midlun/.default'
```
[the remark this answers is not in the document]
Comment (valurefugl) [reply charts/services/services-auth-delegation-api/values.dev.yaml:34]: not secrets

File: charts/services/services-auth-delegation-api/values.staging.yaml [hunk 3]
```diff
@@ -28,6 +28,10 @@ env:
   IDENTITY_SERVER_CLIENT_ID: '@island.is/clients/auth-api'
   IDENTITY_SERVER_ISSUER_URL: 'https://identity-server.staging01.devland.is'
   LOG_LEVEL: 'info'
+  NATIONAL_REGISTRY_B2C_CLIENT_ID: '6cf94113-d326-4e4d-b97c-1fea12d2f5e1'
+  NATIONAL_REGISTRY_B2C_ENDPOINT: 'https://skraidentitydev.b2clogin.com/skraidentitydev.onmicrosoft.com/b2c_1_midlun_flow/oauth2/v2.0/token'
+  NATIONAL_REGISTRY_B2C_PATH: 'IS-TEST/GOV/10001/SKRA-Cloud-Protected/Midlun-v1'
+  NATIONAL_REGISTRY_B2C_SCOPE: 'https://skraidentitydev.onmicrosoft.com/midlun/.default'
```
[the remark this answers is not in the document]
Comment (valurefugl) [reply charts/services/services-auth-delegation-api/values.staging.yaml:34]: not secrets

File: charts/identity-server/values.staging.yaml [hunk 4]
```diff
@@ -323,6 +323,10 @@ services-auth-delegation-api:
     IDENTITY_SERVER_CLIENT_ID: '@island.is/clients/auth-api'
     IDENTITY_SERVER_ISSUER_URL: 'https://identity-server.staging01.devland.is'
     LOG_LEVEL: 'info'
+    NATIONAL_REGISTRY_B2C_CLIENT_ID: '6cf94113-d326-4e4d-b97c-1fea12d2f5e1'
+    NATIONAL_REGISTRY_B2C_ENDPOINT: 'https://skraidentitydev.b2clogin.com/skraidentitydev.onmicrosoft.com/b2c_1_midlun_flow/oauth2/v2.0/token'
+    NATIONAL_REGISTRY_B2C_PATH: 'IS-TEST/GOV/10001/SKRA-Cloud-Protected/Midlun-v1'
+    NATIONAL_REGISTRY_B2C_SCOPE: 'https://skraidentitydev.onmicrosoft.com/midlun/.default'
```
[the remark this answers is not in the document]
Comment (valurefugl) [reply charts/identity-server/values.staging.yaml:329]: not secret

File: charts/services/services-auth-delegation-api/values.prod.yaml [hunk 5]
```diff
@@ -98,6 +102,7 @@
   CONFIGCAT_SDK_KEY: '/k8s/configcat/CONFIGCAT_SDK_KEY'
   DB_PASS: '/k8s/servicesauth/DB_PASSWORD'
   IDENTITY_SERVER_CLIENT_SECRET: '/k8s/services-auth/IDENTITY_SERVER_CLIENT_SECRET'
+  NATIONAL_REGISTRY_B2C_CLIENT_SECRET: '/k8s/services-auth/NATIONAL_REGISTRY_B2C_CLIENT_SECRET'
```
[the remark this answers is not in the document]
Comment (valurefugl) [reply charts/services/services-auth-delegation-api/values.prod.yaml:105]: intended

**Review by peturgq — APPROVED** [review]

**Pull request merged**